app/models/mixins/relationship_mixin.rb

```diff
@@ -682,4 +754,12 @@ def detect_ancestor(*args, &block)
   def puts_relationship_tree
     Relationship.puts_arranged_resources(subtree_arranged)
   end
+
+  private
+
+  # this allows us to call similarly named ancestry or mixin methods
+  def call_ancestry_method(method_name, *args)
+    bound_method = ((@@ancestry_method ||= {})[method_name] ||= ::Ancestry::InstanceMethods.instance_method(method_name)).bind(self)
```

Comment:
wondering if this will conflict with other classes.  Think that maybe a class variable instead of an `@@` may be the best way to go